### PR TITLE
React Metrics webapp

### PR DESCRIPTION
**Pull request by balatstar**
The following have been implemented in this branch:

- React (CRA) + Redux toolkit set up 
- React Router set up
- connection to the UK Carbon Intensity API
- homepage with links to regions categorized by carbon intensity indices
- regions list pages filtered by intensity index
- region detail pages displaying the carbon generation mix for the forecast
- styling based on reference
- responsive layout
- unit and integration tests

Project documentation video: https://www.loom.com/share/0526335d01824d1d9e25b87f9106d0fc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Warm kudos to Microverse for guidance and structure during my learning journey.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+Original design idea by [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
```
Comment (ridwanediallo) [inline README.md]: - [ ] You did a good job of acknowledging the creator of the original design 👍️. However, for the sake of concern separation, I believe it is more practical to place it under the acknowledgment section rather to the license section. Move the text that reads, `Original design idea by Nelson Sakwa`, to the acknowledgement section.👌 😊

File: README.md [hunk 2]
```diff
@@ -1 +1,148 @@
...
+To run tests, run the following command:
+
+npm test
+
+### Deployment
+
+You can deploy this project using:
+
+npm install
+
```
Comment (ridwanediallo) [inline README.md]: - [ ] Please be sure to enter the appropriate information to this section. The `npm install` project command is used to install the project's dependencies.😊

**Review by ridwanediallo — APPROVED** [review]
# Changes Required ♻️
## Hi @balatstar , 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- App design follows the original design. ✔
- The app has successfully been tested.  ✔


 ## Required Changes ♻️

- [ ] Excellent work on the presentational video recording. But given the requirements, I respectfully advise you to record the video again and emphasize a particularly interesting piece of code or feature from your project. 👌 😊

### Requirement :point_down: 

Screenshot from 2023-09-07 20-47-23


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ridwanediallo** [comment]
Hi, @balatstar If you see the message "[ridwanediallo](https://github.com/ridwanediallo) approved these changes," it was an error and I apologize. Hence, please check the changes that have been requested.

**Comment by balatstar** [comment]
I have re-recorded the presentation and updated the link on the PR description. Pasting the video link here also for reference: https://www.loom.com/share/0526335d01824d1d9e25b87f9106d0fc

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchRegions } from '../redux/region/regionSlice';
+import './Home.css';
+
```
Comment (tufoinnkuo10) [inline src/components/Home.js:6]: - [optional] nice job. According to the guidelines and also to make your application easy to use, you were to ensure the Home page has a means of filtering the list of categories using given parameters, e.g. category name.
- [optional] please, your home page has a back arrow which leads to nowhere. Kindly fix that. The back arrows are for the details page, leading back to the home page. Nice work.

Screenshot_23

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @balatstar 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- work well presented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

nice

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**